FaunaDB.Client.Test/TestCase.cs

```diff
@@ -40,21 +42,18 @@ async Task SetUpAsync()
             var domain = Env("FAUNA_DOMAIN", "localhost");
             var scheme = Env("FAUNA_SCHEME", "http");
             var port = Env("FAUNA_PORT", "8443");
-            var secret = Env("FAUNA_ROOT_KEY", "secret");
-            var endpoint = $"{scheme}://{domain}:{port}";
-            var httpClient = new HttpClient();
-            httpClient.BaseAddress = new Uri("https://www.wrongbaseaddress.com");
 
-            rootClient = new FaunaClient(secret: secret, endpoint: endpoint, httpClient: httpClient);
+            faunaSecret = Env("FAUNA_ROOT_KEY", "secret");
+            faunaEndpoint = $"{scheme}://{domain}:{port}";
+            rootClient = new FaunaClient(secret: faunaSecret, endpoint: faunaEndpoint, httpClient: new HttpClient());
```

Comment:
aren't the driver already instantiating `HttpClient` if we do not provide one? if so, you can just remove this argument

Reply:
fixed